packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1663,7 +1663,15 @@ abstract class MultiChildRenderObjectWidget extends RenderObjectWidget {
   /// objects.
   MultiChildRenderObjectWidget({ Key key, this.children = const <Widget>[] })
     : assert(children != null),
-      assert(!children.any((Widget child) => child == null)), // https://github.com/dart-lang/sdk/issues/29276
+      assert(() {
+        final int index = children.indexOf(null);
+        if (index >= 0) {
+          throw AssertionError(
+            'children must not contain any null Widgets, but found one in index $index'
```

Comment:
This looks good. I would maybe change the message to be a little more clearer: 
'The widget's children must not contain null values, but a null value was found at index $index.'

Reply:
Thank you! To tell you the truth I was at a loss of how to start the sentence, but this is good 😄